httphandler/handlerequests/v1/requestshandler.go

```diff
@@ -219,6 +220,13 @@ func (handler *HTTPHandler) Ready(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+// return process heap information
+func (handler *HTTPHandler) Heap(w http.ResponseWriter, r *http.Request) {
```

Comment:
Please use the regular [pprof ](https://pkg.go.dev/net/http/pprof)APIs.

import _ "net/http/pprof"
http.ListenAndServe("localhost:6060", nil)

Reply:
pls see changes